src/js/term/utils.ts

```diff
@@ -86,16 +86,23 @@ export function setDefinitionPosition(
     const fitDefinitionDocument =
         isAlignSwapped && !definitionOutOfScreenOnLeft ? definitionWidth - termWidth : 0;
     const customHeaderTop = getCoords(definitionParent).top - definitionParent.offsetTop;
+    
+    const offsetRight = 5
+    const shiftLeft = definitionOutOfScreenOnRight ? (definitionRightCoordinate - document.body.clientWidth + offsetRight) : 0
+    const leftOffset =
```

Comment:
it would be better to adhere to the same logic for naming variables, you have `offsetRight shiftLeft` and `leftOffset`, I suggest doing it `offestLeft`